### PR TITLE
feat: integrate kapa into website

### PR DESCRIPTION
**Pull request by Gallardot**
We want to try to improve developer experience by integrating kapa into dolphinscheduler website. 

snapshot:

image

### CODE REVIEW

File: src/components/footer/index.jsx [hunk 1]
```diff
@@ -11,6 +12,24 @@ import "./index.scss";
...
   const { locale, t } = useTranslation();
+  useEffect(() => {
+    const script = document.createElement('script');
+
+    script.src = "https://widget.kapa.ai/kapa-widget.bundle.js";
+    script.async = true;
+    script.dataset.websiteId = "e3268e5d-c0f1-4e71-819c-c60ebb2215a7";
+    script.dataset.projectName = "Apache DolphinScheduler";
+    script.dataset.projectColor = "#0097E0";
+    script.dataset.projectLogo = "images/logo_400x400.jpg";
```
Comment (ruanwenjun) [inline src/components/footer/index.jsx:23]: Does the logo already exist under our project?

Comment (Gallardot) [reply src/components/footer/index.jsx:23]: Not yet. This PR will add this logo. Will it be deployed automatically after the PR merge?

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Review by EricGao888 — COMMENTED** [review]
Cool! LGTM

**Pull request merged**